_includes/github.html

```diff
@@ -1,7 +1,7 @@
 {% assign github_branch = site.github.branch | default : "develop" %}
 
 <a href="{{ site.github.repository_url }}/tree/{{ github_branch }}/{{ page.path }}"
-    class="github" target="_blank"
+    id="github" class="github" target="_blank"
```

Comment:
If we add `id="github"` I suppose we can remove `class="github"`?

```suggestion
    id="github" target="_blank"
```